.github/workflows/ci.yml

```diff
@@ -1,19 +1,26 @@
-name: Unit Tests
+name: CI
 
 on:
   push:
-    branches: ["master"]
   pull_request:
-    branches: ["master"]
+  workflow_dispatch:
 
 jobs:
-  test:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: psf/black@21.5b1
```

Comment:
Can we have the same `black` version here as in `pre-commit-config.yaml`?

Reply:
Forgot to replace this with the pre-commit invocation! Thanks for pointing it out.